tests/src/test/scala/quasiquotes/QuasiquoteSuite.scala

```diff
@@ -220,20 +235,46 @@ class QuasiquoteSuite extends FunSuite {
     assert(expr3.show[Syntax] === "b")
   }
 
-//  test("q\"$expr match { ..case $cases }\"") {
-//    val q"$expr match { ..case $cases }" =
-//      q"foo match { case bar => baz; case _ => foo }" // fixme test is broken, so even does not compile
-//  }
+  test("1 q\"$expr match { ..case $cases }\"") {
+    val q"$expr match { case bar => baz; ..case $casez; case q => w}" = q"foo match { case bar => baz; case _ => foo ; case q => w }"
+    assert(expr.show[Syntax] === "foo")
+    assert(casez.toString === "List(case _ => foo)")
+    assert(casez(0).show[Syntax] === "case _ => foo")
+  }
 
-//  test("q\"try $expr catch { ..case $cases } finally $expropt\"") {
-//    val q"try $expr catch { ..case $cases } finally $expropt" = // fixme test is broken, so even does not compile
-//      q"try foo catch { case _ => bar} finally baz"
-//  }
+  test("2 q\"$expr match { ..case $cases }\"") {
+    val q"$expr match { case bar => baz; ..case $casez; case _ => foo }" = q"foo match { case bar => baz; case _ => foo }"
+    assert(expr.show[Syntax] === "foo")
+    assert(casez.isEmpty)
+  }
 
-//  test("q\"try $expr catch $expr finally $expropt\"") {
-//    val q"try $expr catch $expr finally $expropt" =
-//      q"try foo catch { case _ => } finally bar" // fixme test is broken, so even does not compile
-//  }
+  test("3 q\"$expr match { ..case $cases }\"") {
+    val q"$expr match { ..case $casez }" = q"foo match { case bar => baz; case _ => foo }"
+    assert(expr.show[Syntax] === "foo")
+    assert(casez.toString === "List(case bar => baz, case _ => foo)")
+    assert(casez(0).show[Syntax] === "case bar => baz")
+    assert(casez(1).show[Syntax] === "case _ => foo")
+  }
+
+  // todo change to expropt (and test it) after issue #199 resolved
```

Comment:
Could you change the comment to say `TODO: ...` to be in line with other todos in the repo?